src/main/resources/data/brass_armory/recipes/slime_arrow.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "X",
+    "#",
+    "Y"
+  ],
+  "key": {
+    "#": {
+      "item": "minecraft:stick"
+    },
+    "X": {
+      "item": "minecraft:slime_ball"
```

Comment:
Don't we want this to be a slime block or something? With looting small slimes can drop multiple slimeballs making this arrow unlimited if you have the sticks and feathers.

Reply:
Brass and I were talking about this the other day too, and the point I made to him was that if anyone is crazy enough to farm slime using the arrows, then they can go for it because it will never outpace even a small like 2 level slime farm in a slime chunk. 

And realistically, they are really there as another option to a basic arrow which can be farmed relatively easily anyways.  And if we turn down their damage by 1 from 4 to 3, so that they deal less damage than a normal arrow, players probably will use them more in the early game before they have a good way to get regular arrows. It gives players an option for arrows that can be easier than mining flint or setting up a skeleton farm, but that has tradeoffs like less damage and the fact that it spawns a slime.

Reply:
But that's just my view and I'm open to changing it if you've got reasons for why we should xD

Reply:
No it seems fine like this. But I only wanted to point it out if it had been missed. :)